python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -13,6 +13,8 @@
 VPDQ_TIMESTAMP_PRECISION = 3
 VPDQ_QUALITY_THRESHOLD = 50
 VPDQ_DISTANCE_THRESHOLD = 31
+VPDQ_QUERY_MATCH_THRESHOLD_PERCENT = 0.0
+VPDQ_INDEX_MATCH_THRESHOLD_PERCENT = 0.0
```

Comment:
This probably should have a comment to explain that this won't trigger unless there's at least one matching frame (and you should confirm that this is the case in the behavior of the functions).

Reply:
Have added a test for this.